src/components/CustomStatusBar/types.ts

```diff
@@ -0,0 +1,6 @@
+type CustomStatusBar = {
+    (): React.ReactNode;
+    displayName?: string;
```

Comment:
Sorry I didn't get it. Where are these properties used?

Reply:
Maybe this came from `CustomStatusBar.displayName = 'CustomStatusBar';` but we don't really need to type this.
I don't see the need of this file as CustomStatusBar component doesn't have any props

Reply:
Exactly, I had similar thought

Reply:
This is due to guidelines , [here](https://github.com/Expensify/App/pull/29744#issuecomment-1810750551) is the reasoning.

Reply:
We don't have this pattern for any other TS components which have props.
Is it guideline not defining types of component itself if has props?
And defining types of component itself if no props?

Reply:
Okay, why displayName is an optional property?

Reply:
@MonilBhavsar adjusted, `displayName` is not optional now

Reply:
> We don't have this pattern for any other TS components which have props.
Is it guideline not defining types of component itself if has props?
And defining types of component itself if no props?

I think we should add this to our guidelines in .md file